config/initializers/good_job.rb

```diff
@@ -9,7 +9,7 @@
   config.good_job.retry_on_unhandled_error = false
   config.good_job.on_thread_error = ->(exception) { Raven.capture_exception(exception) }
   config.good_job.execution_mode = :external
-  # config.good_job.queues = '*'
+  config.good_job.queues = '-auxiliary, *'
```

Comment:
config.good_job.queues = '-auxiliary, *' will result in:
Two workers
Each running both the -auxiliary and * queues
What we want:
Two workers
One running the -auxiliary queue
One running the * queue